15 php cs fixer issues

badams opened the pull request:
#15 Adding php-cs-fixer config, and changes from running said config

stelgenhof commented on .php_cs:
```diff
@@ -0,0 +1,80 @@
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(__DIR__);
+
+$fixers = array(
```
Sorry I haven't checked myself yet, but does this custom list cover PSR2? I just would like Yasumi to be PSR2 compliant. Any additional ones are fine to me, but not mandatory.

stelgenhof replied:
Reviewed the PR and looking good! Mostly it is formatting/style corrections. However seems these are not necessarily PSR2 corrections, but nonetheless good. Let me merge it this weekend, as I want to check a few little things more

badams replied:
I guess its a starting point, having the config in place means this can eventually be formalized, and the `php-cs-fixer` checks can be added to travis once that is done. 

There are a lot of formatting changes in this PR - I didn't even manage to review them all.

I've just been looking over the PSRs, and Fixers defined for them and this config still includes a lot of symfony cs fixers. I might revert the previous commit and re-run php-cs-fixer with only the PSR based fixers.

badams commented:
@stelgenhof I think I'm happy with this now :)

To summarize the changes in this PR

- I adjusted the list to only include PSR based fixers
- I've also added a shortcut command to composer `composer php-cs-fixer` which runs the fixers.
- I've php-cs-fixer now also runs on the CI server, meaning PRs will need to be PSR2 compliant before they can be merged.

badams commented:
Also, a note about the tests directory and namespaces, it appears to be part of PSR4

> The contiguous sub-namespace names after the "namespace prefix" correspond to a subdirectory within a "base directory", in which the namespace separators represent directory separators. The subdirectory name MUST match the case of the sub-namespace names.

Which is a requirement of PSR2 as far as I understand it

stelgenhof commented:
I think the contiguous sub-namespace requirements was the culprit I believe. Let me check once more and merge your PR

stelgenhof commented:
Would you mind changing the "tests" folder back to uppercase? I prefer the sub-namespaces to be consistent. If not, I can do it after merging the PR :)

As for `- ./vendor/bin/php-cs-fixer --diff --dry-run -v fix`, you said it will run on the CI server however this is just a dry-run. Any reason for that?

badams commented:
@stelgenhof would you want to change the actual directory from `tests` to `Tests`?  at least that way it does align still align with PSR4

stelgenhof commented:
Ah indeed. Then I would leave it as you did. No need to change. I will merge it

badams commented:
Alright, go ahead. I'll need to update my other pull request to meet the new CS requirements :)
